docs/rhoas/iac_tools_rhoas/README.adoc

```diff
@@ -0,0 +1,703 @@
+[discrete]
+=== Creating a Kafka instance
+
+[role="_abstract"]
+To manage a Kafka instance using Terraform, use the `rhoas_kafka` resource.
+
+.Prerequisites
+
+* You have an offline token that is accessible to Terraform.
+* You have a Red Hat account.
+
+.Procedure
+
+. Open your IDE of choice and enter the example rhoas_kafka resource. This example uses the “instance”  identifier and creates a Kafka instance called `my-instance`. The only required fields are the `name`, `billing_model` and `plan` fields. The following values are set by default when you run `terraform apply`. All other information for the instance is provided by the Kafka APIs.
++
+.. `cloud provider`: A list of available cloud providers can be obtained using `data.rhoas_cloud_providers`.
+.. `region`: A list of available regions can be obtained using
+`data.rhoas_cloud_providers_regions`
+
++
+.Example `rhoas_kafka` resource
+[source,shell]
+----
+$ terraform {
+  required_providers {
+    rhoas = {
+      source  = "registry.terraform.io/redhat-developer/rhoas"
+      version = "0.3.0"
+    }
+  }
+}
+
+provider "rhoas" {}
+
+resource "rhoas_kafka" "my-instance" {
+  name = "my-instance"
+}
+
+output "bootstrap_server_my-instance" {
+  value = rhoas_kafka.my-instance.bootstrap_server_host
+}
+----
+. Open a terminal and initilize Terraform by entering the following command.
++
+[source,shell]
+----
+$ terraform init
+----
+. Apply the changes by entering the following command.
++
+[source,shell]
+----
+$ terraform apply
+----
+. Terraform displays a message that `rhoas_kafka.my-instance` will be created. Enter *yes*.
+. Verify that the instance has been created in the the table on the *Kafka Instances* page of the {product-kafka} {service-url-kafka}[web console^].
+
+[id="proc-creating-service-account-terraform_{context}"]
+[discrete]
+=== Creating a service account
+
+[role="_abstract"]
+After creating your Kafka instance, you create a service account. To connect your applications or services to a Kafka instance in {product-kafka}, you must first create a service account with credentials.
+
+Use the `rhoas_service_account` resource to manage a service account. Terraform populates the credentials when the resource is applied. This example uses the ‘srvaccnt’ identifier and creates a service account called my-service-account.
+
+.Prerequisites
+
+* You have an offline token that is accessible to Terraform.
+* You have a running Kafka instance in {product-kafka}.
+
+.Procedure
+
+. In your IDE, enter the example `rhoas_service_account` resource to create a service account.
++
+
+.Example `rhoas_service_account` resource
+[source,shell]
+----
+$ resource "rhoas_service_account" "my-instance" {
+  name        = "my-service-account"
+  description = "<description of service account>"
+}
+----
+. Apply the changes by entering the following command. Note that you do not need to run `terraform init` if following on from the Create a Kafka instance step.
```

Comment:
I'd add the sentence beginning "Note that..." as it's own step, before this one. Something along these lines:

```
. If you did not already initialize Terraform in your current terminal session, initialize it.
+
[source,shell]
----
$ terraform init
----